### DISCUSSION TITLE
Qodana Cloud Testing

### DISCUSSION THREAD

**Discussion by alfkonee** [JetBrains/Qodana#136] [Q&A] [answered]
Hey there  I found this site 
https://qodana.cloud/
Signed Up and created a Project
I have the token but now I have no idea what do to with it. The Documentation mentions and analysis id  here [Here](https://www.jetbrains.com/help/qodana/2022.3/qodana-jvm-docker-techs.html#docker+config+options+qodana+cloud)
![image](https://user-images.githubusercontent.com/3909521/206218916-6df1f8f5-c89f-4b63-a1f7-22203afd61a6.png)

SO then I guess My question is in two folds 
1. Where do I get the Analysis Id of the Project 
2. how do I integrate the Token Into the Linter

**Comment by shyim** [comment]
It's released now and anything documented :)

**Comment by brichbash** [comment, marked as answer]
Hello,

It seems you found Qodana Cloud before we [officially announced](https://blog.jetbrains.com/qodana/2022/12/preview-for-qodana-cloud/) its preview :)
As @shyim noted, we updated our documentation for Cloud today. If you have any questions, feel free to ask them here on GitHub Discussions or write us at qodana-support@jetbrains.com.

In order to send reports to Qodana Cloud you need to pass `QODANA_TOKEN` to the linter. You can find more detailed information in [our documentation](https://www.jetbrains.com/help/qodana/cloud-forward-reports.html).

> Where do I get the Analysis Id of the Project

Analysis id is an id generated by our linters. It is can be used to overwrite an existing report on Qodana Cloud. At the moment it is only possible to look it up in logs (`log/idea.log`) which is not very convenient. We'll fix that with the next update.
